webpack.config.js

```diff
@@ -94,8 +94,8 @@ if (analyze) {
 }
 
 const optimization = {
-    minimize: true,
-    minimizer: [new TerserPlugin({
+    minimize: debug,
```

Comment:
Should this not be inverted? we want to minimize when debug is false

Reply:
resolved, minimize on prod only, nice catch! thanks!